libcst/codemod/_cli.py

```diff
@@ -251,16 +251,18 @@ def _execute_transform(  # noqa: C901
             scratch={},
         )
 
-        # attempt to work out the module and package name for this file
-        module_name_and_package = calculate_module_and_package(
-            config.repo_root, filename
-        )
-        if module_name_and_package is not None:
+        # determine the module and package name for this file
+        try:
+            module_name_and_package = calculate_module_and_package(
+                config.repo_root or ".", filename
+            )
             transformer.context = replace(
                 transformer.context,
                 full_module_name=module_name_and_package.name,
                 full_package_name=module_name_and_package.package,
             )
+        except ValueError:
+            pass
```

Comment:
Is this something we should at least log about?

Reply:
Good idea